syntax/gitmessengerpopup.vim

```diff
@@ -5,6 +5,7 @@ endif
 syn match gitmessengerHeader '\_^ \%(History\|Commit\|Date\|Author\|Committer\):' display
 syn match gitmessengerHash '\%(\<Commit: \)\@<=[[:xdigit:]]\+' display
 syn match gitmessengerHistory '\%(\<History: \)\@<=#\d\+' display
+syn match gitmessengerEmail '\%(\<\(Author\|Committer\): .*\)\@<=<.\+>' display
```

Comment:
Please add `\_^ ` to ensure this syntax highlighting matches to headers.

```suggestion
syn match gitmessengerEmail '\_^ \%(\%(Author\|Committer\): .*\)\@<=<.\+>' display
```

Reply:
Actually I didn't intend to match the headers themselves here. The author line may look like this for example:

```
 Author: John Doe <john.doe@mail.com>
```

I just want to match `<john.doe@mail.com>` in this case. This is similar to matching the hash and the history number.

If we want to match `\_^ `, it should probably be done inside the parenthesis, for all cases (so for `gitmessengerHash` and `gitmessengerHistory` as well). I'll make another pull-request for that if you wish :slightly_smiling_face:

Reply:
But I missed a `%`.